specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2239 @@
+        "summaryAspects"
+      ],
+      "properties": {
+        "summaryAspects": {
+          "type": "string"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskParameters"
+        },
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "AnalyzeConversationSummarizationResult": {
+      "type": "object",
+      "description": "Result for the summarization task on the conversation.",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/SummaryResult"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobResult"
+        }
+      ],
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "ConversationalSummarizationResults"
+    },
+    "SummaryResult": {
+      "type": "object",
+      "required": [
+        "conversations"
+      ],
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ConversationsSummaryResult"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ConversationResultBase"
+        }
+      ]
+    },
+    "ConversationsSummaryResult": {
+      "type": "object",
+      "required": [
+        "summary"
+      ],
+      "properties": {
+        "summaries": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/SummaryResultItem"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "SummaryResultItem": {
+      "type": "object",
+      "properties": {
+        "aspect": {
+          "type": "string"
+        },
+        "text": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "aspect",
+        "text"
+      ]
+    },
+    "AnalyzeConversationJobState": {
+      "description": "Contains the status of the analyze conversations job submitted along with related statistics.",
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/JobState"
+        },
+        {
+          "$ref": "#/definitions/ConversationTasksState"
+        },
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobStatistics"
+        }
+      ]
+    },
+    "AnalyzeConversationJobStatistics": {
+      "description": "Contains the statistics for the job submitted.",
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/ConversationRequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "ConversationTasksState": {
+      "description": "Contains the state for the tasks being executed as part of the analyze conversation job submitted.",
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
```

Comment:
Here and on other result (return-only) properties, can you add `"readonly": true`? This will facilitate better SDK code generation as well. It wouldn't make sense, for example, to have setters on read-only properties.